VimeoNetworking/Sources/ResponseCache.swift

```diff
@@ -150,7 +150,7 @@ final internal class ResponseCache
                 let fileManager = NSFileManager()
                 
                 let directoryURL = self.cachesDirectoryURL()
-                let fileURL = self.fileURLForKey(key: key)
+                _ = self.fileURLForKey(key: key)
```

Comment:
It wasn't, no. And now that I look at this again I can see that just below here we have
```Swift
guard let directoryPath = directoryURL.path,
    let filePath = self.fileURLForKey(key: key)?.path
else
```
So I think maybe this was a mistake. I'm going to remove the line.